src/main/java/seedu/address/model/event/Location.java

```diff
@@ -15,7 +15,7 @@ public class Location {
      * The first character of the location must not be a whitespace,
      * otherwise " " (a blank string) becomes a valid input.
      */
-    public static final String VALIDATION_REGEX = "[A-Za-z0-9 _.,!\"'/$]+";
+    public static final String VALIDATION_REGEX = "^[a-zA-Z0-9 !\\\"#$%&'()*+,-./:;<=>?@[\\\\]^_` ]+$";
```

Comment:
try this regex: ```^[a-zA-Z0-9 !\\\"#$%&'()*+,-./:;<=>?@\[\\\\]^_` ]+$```

Reply:
this one also cannot leh, more buggy than the original regex actually

Reply:
oof. i think should just use `.` - https://stackoverflow.com/questions/2912894/how-to-match-any-character-in-regular-expression